raiden/tests/integration/test_matrix_transport.py

```diff
@@ -302,6 +309,72 @@ def test_matrix_message_sync(
     transport1.get()
 
 
+@pytest.mark.parametrize('number_of_nodes', [2])
+@pytest.mark.parametrize('channels_per_node', [1])
+@pytest.mark.parametrize('number_of_tokens', [1])
+def test_matrix_tx_error_handling(
+        skip_if_not_matrix,
+        raiden_chain,
+        token_addresses,
+):
+    """ Test exception propagation for transactions triggered by the transport greenlet.
+    """
+    app0, app1 = raiden_chain
+    token_network = views.get_token_network_by_token_address(
+        views.state_from_app(app0),
```

Comment:
nit: use kwargs here and in the rest of the function calls in the test that don't have them